src/components/ui/ListedValue.tsx

```diff
@@ -0,0 +1,66 @@
+import { truncateMiddleIfNeeded } from '@big-whale-labs/frontend-utils'
+import Cross from 'icons/Cross'
+import classnames, {
+  alignItems,
+  backgroundColor,
+  borderRadius,
+  cursor,
+  display,
+  gap,
+  height,
+  padding,
+  textColor,
+  width,
+} from 'classnames/tailwind'
+import useBreakpoints from 'hooks/useBreakpoints'
+
+const valueWrapper = (alternativeStyle?: boolean, isDifferent?: boolean) =>
+  classnames(
+    textColor({ 'text-primary-dark': isDifferent }),
+    display('flex'),
+    alignItems('items-center'),
+    gap('gap-x-1'),
+    backgroundColor(
+      isDifferent
+        ? 'bg-secondary'
+        : alternativeStyle
+        ? 'bg-primary-semi-dimmed'
+        : 'bg-primary-dimmed'
```

Comment:
Maybe we can move it to the const instead of using ternary operator inside ternary operator

Reply:
Used object, thanks :)